virtual_ecosystem/models/hydrology/hydrology_tools.py

```diff
@@ -107,14 +89,18 @@ def setup_hydrology_input_current_timestep(
         seed=seed,
     )
 
+    # named 'surface_...' for now TODO needs to be replaced with 2m above ground
+    # VIVI - these variables are explicitly 1 dimensional for a single layer. They come
+    # out of the subset as 2D (1, n_cells) and so I've squeezed them.
+    # squeezed them
     for out_var, in_var in (
```

Comment:
That's fine with me, I think I had misunderstood how exactly the new layer_structure works

Reply:
All of the indices are numpy arrays, which leads to 2D slices. That's not obvious and needs documenting better. This can be done more cleanly by converting the index to a scalar where the index is know to be to a single layer.